auto_nag/scripts/inactive_ni_pending.py

```diff
@@ -0,0 +1,184 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from dateutil.relativedelta import relativedelta
+from libmozdata import utils as lmdutils
+
+from auto_nag import utils
+from auto_nag.bzcleaner import BzCleaner
+from auto_nag.user_activity import UserActivity
+
+# TODO: should be moved when resolving https://github.com/mozilla/relman-auto-nag/issues/1384
+HIGH_PRIORITY = {"P1", "P2"}
+HIGH_SEVERITY = {"S1", "critical", "S2", "major"}
+
+
+class InactiveNeedinfoPending(BzCleaner):
+    def __init__(self):
+        super(InactiveNeedinfoPending, self).__init__()
+        self.max_actions = utils.get_config(self.name(), "max_actions", 7)
+        self.__recent_bug_limit = lmdutils.get_date_ymd("today") - relativedelta(
```

Comment:
Nit: this could be a constant defined at the top of the file

Reply:
I defined it in the class so we can easily make it configurable if needed. I switch to define it as a constant in 4fe1239c1f1bdf65130ae479e0f7555ccf679080. We could revert the commit if you think we need it to be configurable.